src/scripts/combineSplitFilesToMakeALargeDatasetFiles-CSC.ts

```diff
@@ -22,60 +28,119 @@ main.map((item: any, index: number) => {
 	return item;
 });
 
-fs.writeFileSync(path.join(__dirname, '../', 'assets/country.json'), JSON.stringify(allCountries, null, 3));
+fs.writeFileSync(path.join(__dirname, '../', 'assets/allCountryNew.json'), JSON.stringify(allCountries, null, 3));
+
+const countryMetas: { [Property: string]: { countryPath: string } } = allCountries.reduce((accumulator, country) => {
+	accumulator[country.isoCode] = { countryPath: `${country.name.replace(/\W/g, '_')}-${country.isoCode}` };
+	return accumulator;
+}, {} as { [Property: string]: { countryPath: string } });
 
 /**
  * State
  */
-const allStates: any[] = [];
-const alStatesNestedLite = JSON.parse(
-	fs.readFileSync(path.join(__dirname, PATH_TO_DATA_FOLDER, '/allStatesNested.lite.json'), 'utf-8'),
-);
-const alStatesNestedGeo = JSON.parse(
-	fs.readFileSync(path.join(__dirname, PATH_TO_DATA_FOLDER, '/allStatesNested.geo.json'), 'utf-8'),
-);
+const allStates: IState[] = [];
+allCountries.forEach((country) => {
+	process.stdout.write(`Combining ${country.name}-${country.isoCode}...`);
+	const allStatesLite = new Map<string, IState>();
+	const allStatesGeo = new Map<string, IState>();
+	try {
+		const { countryPath } = countryMetas[country.isoCode]; // `${country.name.replace(/\W/g, '_')}-${country.isoCode}`;
+		(JSON.parse(
+			fs.readFileSync(path.join(__dirname, PATH_TO_DATA_FOLDER, countryPath, 'allStates.lite.json'), 'utf-8'),
+		) as IState[]).reduce((accumulator, item) => {
+			accumulator.set(item.isoCode, item);
+			return accumulator;
+		}, allStatesLite);
```

Comment:
prettier issue in my vscode

Reply:
![Screen Shot 2023-04-02 at 6 59 04 PM](https://user-images.githubusercontent.com/2958292/229355862-dd847628-b040-4ef0-9765-32d689f4dd10.png)

Reply:
Can you please tell me what prettier plugin you use? I'll use the same next time.

Reply:
applying prettier once again fix the issue :-)